sdk/identity/azure-identity/images/mermaidjs/DefaultAzureCredentialAuthFlow.md

```diff
@@ -22,7 +22,7 @@ flowchart LR;
     %% Add API ref links to credential type boxes
     click A "https://docs.microsoft.com/java/api/com.azure.identity.environmentcredential?view=azure-java-stable" _blank;
     click B "https://docs.microsoft.com/java/api/com.azure.identity.managedidentitycredential?view=azure-java-stable" _blank;
-    click C "https://docs.microsoft.com/java/api/com.azure.identity.intellijcredential?view=azure-java-stable" _blank;
+    click D "https://docs.microsoft.com/java/api/com.azure.identity.intellijcredential?view=azure-java-stable" _blank;
```

Comment:
While you're in here, let's replace "docs.microsoft.com" with "learn.microsoft.com" for each of these links.

Reply:
Oh, good catch.